ingestion-beam/src/test/java/com/mozilla/telemetry/transforms/PubsubMessageToTableRowTest.java

```diff
@@ -136,13 +136,13 @@ public void testAdditionalProperties() throws Exception {
     parent.put("outer", new HashMap<>(ImmutableMap.of("otherField", 3, //
         "mapField", new HashMap<>(ImmutableMap.of("foo", 3, "bar", 4)))));
     parent.put("clientId", "abc123");
-    parent.put("otherStrangeIdField", 3);
+    parent.put("otherStrangeIDField", 3);
```

Comment:
this change seems unrelated to the scope, just cleaning up along the way?

Reply:
I think I made this change to exercise another edge case of snake casing, but in this case the expected behavior is that the field should be left unchanged and put in additional_properties. I may go ahead and revert it.